EnglishFilter/sections/whitelist_stealth.txt

```diff
@@ -42,6 +42,8 @@
 !
 !
 !
+! bcee.snet.lu - DPI feature of the Adguard Stealth break the website
+@@||bcee.snet.lu^$stealth
```

Comment:
It should be reported here https://github.com/AdguardTeam/CoreLibs/issues/1645

Reply:
`$stealth` exclusion also should be added, and mentioned in CL issue(disable Base filter before check)

Reply:
`DPI feature of the Adguard Stealth break the website`
I think it's better to be
```
[Stealth Mode - Protect from DPI]
```

Reply:
Also add other domains from the task https://github.com/AdguardTeam/CoreLibs/issues/1645